src/app/modules/test-navigator/test-navigator.component.ts

```diff
@@ -66,6 +66,14 @@ export class TestNavigatorComponent implements OnInit, OnDestroy {
     indicatorFields: []
   };
 
+  getFilteredOutMarkers = (type: FilterType) => {
+    let markers = ValidationMarkerSummary.zero;
+    if (this.filterState && !this.filterState[type] && (this.filterState.aml || this.filterState.tsl || this.filterState.tcl)) {
```

Comment:
would be more readable if e.g. aml || tsl || ... be captured into const hasActiveFilter or something like that